drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/case_note_export.rb

```diff
@@ -0,0 +1,62 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: false
```

Comment:
This can probably be `true` for a new file (if not, we should adjust the string/array logic.